src/components/case_manager/NavBar.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styles from "../../styles/Navbar.module.css";
+import Link from "next/link";
+import Image from "next/image";
+import { BsPersonWorkspace, BsPeople, BsGear } from "react-icons/bs";
+import { FiLogOut } from "react-icons/fi";
+import { signOut } from "next-auth/react";
+import { trpc } from "@/client/lib/trpc";
+
+interface Props {
+  children: React.ReactNode;
+}
+const NavBar: React.FC<Props> = ({ children }) => {
+  const { data: me } = trpc.user.getMe.useQuery();
```

Comment:
While sitting on the sign in page, a `401: Unauthorized error` toast pops up because its trying to use getMe without being logged in. Is there a way to prevent this? I think one possibility is to create a new backend procedure for this.

Reply:
Currently investigating our options

Reply:
We can create a separate issue for this, in the spirit of faster and smaller PRs.